client/state/ui/section/actions.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { SECTION_SET } from 'state/action-types';
+
+export function setSection( section, options = {} ) {
+	options.type = SECTION_SET;
```

Comment:
Mutating the `options` argument is rather ugly ☹️ We have an opportunity to clean this up here.

Reply:
That's a very good point. Fixed! No more unexpected arg mutation 🙂

Reply:
Thanks for fixing! I like the result :+1: 🙂